src/lang.rs

```diff
@@ -246,6 +246,11 @@ pub enum InlineOperator {
         /// check the value to see if it's sane or provide a default if no number was given.
         count: Option<Positioned<f64>>,
     },
+    Split {
+        separator: String,
+        input_column: Option<String>,
```

Comment:
use `Option<Expr>` -- keeping this as a full expression ensures things like `split from row.foo[5]` will work properly -- you can keep it as `Option<Expr>` the entire way down the chain into the actual operator

Reply:
👍 Done!